src/library/scala/collection/View.scala

```diff
@@ -188,7 +188,7 @@ object View extends IterableFactory[View] {
 
   /** A class that splits an underlying collection into two views */
   @SerialVersionUID(3L)
-  class PartitionWith[A, A1, A2](val underlying: SomeIterableOps[A], val f: A => Either[A1, A2]) extends Serializable {
+  class partitionMap[A, A1, A2](val underlying: SomeIterableOps[A], val f: A => Either[A1, A2]) extends Serializable {
```

Comment:
Our naming convention is to start type names with an upper case letter.

Reply:
Oops... search & replace mishap 😅